### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed all references to windows phone  as it is no longed supported platform

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7878

File: microsoft-365/admin/basic-mobility-security/overview.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ You can manage and secure mobile devices when they're connected to your Microsof
 
 ## What types of devices can you manage?
 
-You can use Basic Mobility and Security to manage many types of mobile devices like Windows Phone, Android, iPhone, and iPad. To manage mobile devices used by people in your organization, each person must have an applicable Microsoft 365 license and their device must be enrolled in Basic Mobility and Security.
+You can use Basic Mobility and Security to manage many types of mobile devices like  Android, iPhone, and iPad. To manage mobile devices used by people in your organization, each person must have an applicable Microsoft 365 license and their device must be enrolled in Basic Mobility and Security.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/basic-mobility-security/overview.md]:
```suggestion
You can use Basic Mobility and Security to manage many types of mobile devices like Android, iPhone, and iPad. To manage mobile devices used by people in your organization, each person must have an applicable Microsoft 365 license and their device must be enrolled in Basic Mobility and Security.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**